public/components/marketing-forms/5.html

```diff
@@ -0,0 +1,67 @@
+<!--
+  Heads up! 👋
+
+  Plugins:
+    - @tailwindcss/forms
+-->
+
+<section class="w-full space-y-4 bg-white md:space-y-6">
```

Comment:
Why `w-full`?

Reply:
There should be a container in here and not in the MD file. In it's current state the component is usable as it will render completely differently for a user.

Something like:

```html
<section>
  <div class="max-w-screen-xl px-4 py-8 mx-auto sm:py-12 sm:px-6 lg:px-8">
```

Reply:
I forgot to remove the `w-full`. I'll also remove the container property in the md file